package.json

```
@@ -22,6 +22,7 @@
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
+    "prepack": "chmod +x ./scripts/prepack.sh",
```

chmod only needs to be run once locally to set the file permissions correctly. The file permissions are stored in git.

This script should just invoke the prepack script.
```suggestion
    "prepack": "./scripts/prepack.sh",
```

---

done [here](https://github.com/MetaMask/eth-hd-keyring/pull/73/commits/84cc6cee6a07e2661bc997d4390ae38837d79acd)